OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -535,6 +542,11 @@ void Manager::
 setIntegrator(SimTK::Integrator& integrator) 
 {   
     if (_integ.get() == &integrator) return;
+    if (_timeStepper) {
```

Comment:
Not sure if  throwing here is the ideal behavior. Instead if you reset `_timeStepper` then `integrate` should throw that it has no `TimeStepper` and to call `initializeTimeStepper` first, or you could `setIntegrator()` do that for you after the reset.